### ISSUE TITLE
Publish as dotnet cli tool

### ISSUE THREAD

**Issue by nojaf** [fsprojects/Mechanic#88]
It be nice if Mechanic was available as `clitool`.
Running `dotnet mechanic` would format the `fsproj` in the current folder.
Or show a help page if no (single) fsproj is found in the current directory.
This way other editors than vscode could easily integrate Mechanic.

**Comment by forki** [comment]
marked it up-for-grabs. /cc @enricosada for mentoring

**Comment by JacobChang** [comment]
I'd like to solve this one, Do I need to create a new project or I can just modify Mechanic.CommandLine?

**Comment by forki** [comment]
Whatever you think is best

**Comment by nojaf** [comment]
Could this be published on nuget?

**Comment by forki** [comment]
yes but someone needs to send PR with release process in build.fsx

**Comment by sparebytes** [comment]
I'm new to F# and it's ecosystem. I'm attempting to add a "PublishToNuget" target but I'm having dependency/paket issues. Maybe it's because I have dotnet core 3 preview installed.

As soon as I add `nuget Fake.DotNet.Nuget prerelease` to the `paket.dependencies` and run `./.paket/paket.exe install`, the lock file is drastically changed. Any attempts to run `fake build` thereafter fail. Even if I remove the dependency and re-run paket it still fails.

Then I removed all the `prelease`  from all the dependencies and was able to get the `build.fsx` working after fixing deprecations but I'm still not able to `open Fake.DotNet.Nuget` without `Error FS0039: The namespace 'Nuget' is not defined.`

https://github.com/sparebytes/Mechanic

**Comment by jindraivanek** [comment]
@sparebytes long time that packages was update here. `prerelease` should be definitely removed, it is there because FAKE 5 was in prerelease back then.

Ofc, best thing will be to update whole project to new packages, but there will be for sure bunch of problems..

I think best way forward for now is to update just `Build` group of packages with `paket update -g Build`.

See https://github.com/fsprojects/Mechanic/compare/master...jindraivanek:update where I did that + few other minor things. Its building for me.
